includes/abstracts/abstract.llms.post.model.php

```diff
@@ -402,13 +339,108 @@ public function export() {
 	}
 
 	/**
-	 * Getter
-	 * @param  string $key  property key
+	 * Private getter.
+	 *
+	 * @since [version]
+	 *
+	 * @param string  $key The property key.
+	 * @param boolean $raw Optional. Whether or not we need to get the raw value. Default false.
 	 * @return mixed
+	 */
+	private function ___get( $key, $raw = false ) {
+
+		// force numeric id and prevent filtering on the id.
+		if ( 'id' === $key ) {
+
+			return $raw ? $this->$key : absint( $this->$key );
```

Comment:
I think that the ID should always be forced to an `int` I don't see any reason why we'd want a potentially non int id. Do you?

Reply:
nope, just for consistency :D 
udating...